### PR TITLE
More information for disconnect packet + transport error fix

### PR DESCRIPTION
**Pull request by kpxyy**
This can easily help some people why their skribbler bot is randomly disconnecting

### CODE REVIEW

File: src/client.js [hunk 1]
```diff
@@ -71,6 +71,8 @@ class Client extends events {
 
 		let disconnectReason;
 		let joinErr;
+		let wsReason;
```
Comment (MrDiamond64) [inline src/client.js:74]: Why

File: src/client.js [hunk 2]
```diff
@@ -79,12 +81,15 @@ class Client extends events {
 			disconnectReason = data;
 		});
 
-		socket.on("disconnect", () => {
+		socket.on("disconnect", (data) => {
+			wsReason = data;
```
Comment (MrDiamond64) [inline src/client.js:85]: Why

Comment (kpxyy) [reply src/client.js:85]: to log the disconnect reason provided by skribbl.io

File: src/client.js (hunk 1 above)
Comment (kpxyy) [reply src/client.js:74]: to log the disconnect reason provided by skribbl.io

**Comment by MrDiamond64** [comment]
Superseded by https://github.com/Skribblers/skribbler/commit/b73108023b99c78ab562908e72c8c3d9b763b09a. Create a new PR with the transport fix.

**Pull request closed without merging**